src/goto-cc/Makefile

```diff
@@ -57,6 +57,9 @@ endif
 
 ###############################################################################
 
+goto-gcc$(EXEEXT): goto-cc$(EXEEXT)
+	@ln -sf goto-cc$(EXEEXT) goto-gcc$(EXEEXT)
```

Comment:
I think make has `$(LN)` to improve portability, and I'd suggest not to hide this output, i.e., please remove the `@` - simply because we don't do quiet output elsewhere either.